src/components/calcite-flow-item/calcite-flow-item.e2e.ts

```diff
@@ -203,13 +203,8 @@ describe("calcite-flow-item", () => {
 
   it("should be accessible", async () =>
     accessible(`
-<<<<<<< HEAD
       <calcite-flow-item heading="hello world" summary="test" menu-open show-back-button>
         <div slot="menu-actions">
```

Comment:
@asangma can you put this back to `<div slot="${SLOTS.menuActions}">`

Reply:
Oops!

Reply:
Updated.